COSC481W-2024Fall/MLite
# Set up server to connect to SQS queue and train models

This PR sets up a server that connects to the AWS SQS queue, polls the queue for messages, and processes them one by one to train machine learning models.

**Steps to Test:**

1. Ensure AWS credentials are set up for access to SQS.
2. Run the server to start polling the SQS queue: python3 app2.py
3. The output of the messages that are queued in the sqs will be polled 

Issue #47
